### PR TITLE
⚒️ account avaible balance 

### PR DESCRIPTION
**Pull request by roiLeo**
It will now show avaibleBalance in Teleport/Transfer/CreateCollection

### PR type

- [x] Bugfix
- [x] Feature
- [x] Refactoring
- [x] related with #2292

### CODE REVIEW

**Comment by chandradot99** [comment]
@roiLeo you have done some changes in this PR which also needs to be changed in #2292 , should I merge your changes into mine or should I wait for this PR to get merged and then add new changes accordingly.

**Comment by roiLeo** [comment]
> @roiLeo you have done some changes in this PR which also needs to be changed in #2292 , should I merge your changes into mine or should I wait for this PR to get merged and then add new changes accordingly.

I'd rather wait for the review this could introduce some breaking change

File: utils/api/Query.ts [hunk 1]
```diff
@@ -18,11 +18,8 @@ class Query {
     api: ApiPromise,
     accountId: string
   ): Promise<string> {
-    const cb = api.query.system.account
-    const arg = accountId
-    const balance = await cb(arg)
-    const accountData = (balance as any).data.free.toString()
-    return accountData
+    const { availableBalance } = await api.derive.balances.all(accountId)
+    return availableBalance.toString()
```
Comment (vikiival) [inline utils/api/Query.ts:22]:
```suggestion
    const { freeBalance } = await api.derive.balances.all(accountId)
    return freeBalance.toString()
```

**Review by vikiival — APPROVED** [review]
This is my personal preference but your code is also legit

**Comment by vikiival** [comment]
Screenshot 2022-02-16 at 11 39 00

**Comment by yangwao** [comment]
less opinionated code 🥺😅 

but yeah free is in this context way profound

File: utils/api/Query.ts (hunk 1 above)
Comment (roiLeo) [reply utils/api/Query.ts:22]: What's the difference between freeBalance & availableBalance?

**Pull request merged**